accounts/migrations/0042_contact.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.0.3 on 2021-03-12 18:09
```

Comment:
No creo que valga la pena perder tiempo en esto, pero pareciera que las dos migraciones anteriores (40 y 41) son inútiles. Luego, esta (la 42) parece ser la final pero se llama `0041_delete_contact` a pesar de hacer todo lo contrario (crear el modelo).

Te lo comento porque me pareci[o raro, pero creo que lo podemos dejar así.

Reply:
Eso paso mientras yo hacia mis migrates, no me puse a buscar como cambiarlo porque el efecto sera el mismo